objects/UniversalBenefit.js

```diff
@@ -144,9 +146,9 @@ export function CreateBenefitObjWithCPPData(cppBenefitData) {
     [
       CreateBenefitSummary(
         SummaryTypes.PaymentAmount,
-        cppBenefitData.netAmount
+        '$' + cppBenefitData.netAmount
       ),
-      CreateBenefitSummary(SummaryTypes.NextPayment, nextPayment),
+      CreateBenefitSummary(SummaryTypes.NextPayment, formatDate(nextPayment)),
```

Comment:
While this might seem simpler, I believe the formatting/design should be managed in the component. This requires us to format the date each time we load it into the object, which could happen many times. Alternatively formatting it when it's loaded in the component, means we only need to call the "formatDate" once.
It's probably going to require a Switch/Case on the SummaryType to get the right format for the type of data.

Reply:
Adjusted the font-weight by summaryType, don't think there are any more changes to text size/colour based on the summaryType in the current iteration of the master components but if there's anything missing let me know.